drivers/entropy/entropy_cc310.c

```diff
@@ -74,7 +72,7 @@ static int entropy_cc3xx_rng_get_entropy(
 			 * using the CTR_DRBG features of the
 			 * nrf_cc310_platform/nrf_cc312_platform library.
 			 */
-			res = nrf_cc3xx_platform_ctr_drbg_get(&ctr_drbg_ctx,
+			res = nrf_cc3xx_platform_ctr_drbg_get(NULL,
```

Comment:
Would be nice with a little "NULL means use internal context" comment here since it looks so jarring.

Reply:
I will also add that!